### PR TITLE
feature: add freelancer create profile page 2

### PR DESCRIPTION
**Pull request by braulioportela79**
Second Page for Freelancer user create profile. 

freelancercreateprofilepage2a

freelancercreateprofilepage2b

freelancercreateprofilepage2c

### CODE REVIEW

File: apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+	ThemeBackground,
+	Dashboard,
+	StyledTitle,
+	StyledButton,
+} from "@freelance/components";
+import { ThemeProvider } from "styled-components";
+
+export function CreateProfile2() {
+	const { t } = useTranslation();
+	const [educationList, setEducationList] = useState([{ education: "" }]);
```
Comment (lyudad) [inline apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx]: better empty array

File: apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+	const [workExperienceList, setWorkExperienceList] = useState([{ workExperience: "" }]);
```
Comment (lyudad) [inline apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx]: better empty array

File: apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx (hunk 1 above)
Comment (braulioportela79) [reply apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx]: fixed

File: apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx (hunk 2 above)
Comment (braulioportela79) [reply apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx]: fixed

File: apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+		institute: "",
+		occupation: "",
+		period: "",
+	};
+
+	const workExperience = {
+		company: "",
+		workPosition: "",
+		period: "",
+	};
```
Comment (lyudad) [inline apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx]: no no no, never constants in component. initialisation of this constant will be with each rerender, its very very dab practice, move above the component

File: apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+	const handleAddWorkExperience = () => {
+		setWorkExperienceList(prev => [...prev, workExperience]);
+	};
+
+	const handleRemoveWorkExperience = (index: number) => {
+		const list = [...workExperienceList];
+		list.splice(index, 1);
+		setWorkExperienceList(list);
+	};
```
Comment (lyudad) [inline apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx]: too many logic, move to custom hook

File: apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+
+	return (
+		<ThemeProvider theme={ThemeColors && ThemeBackground}>
```
Comment (lyudad) [inline apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx]: it should be once in the root file, no?

File: libs/components/src/lib/dashboard-menu/dashboard-menu.tsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+	userRole: "freelancer";
+}
+
+export function DashboardMenu({ userRole }: DashboardMenuProps) {
+	const { t } = useTranslation();
+
+	return (
+		<Container>
+			<img id="logo" src="/assets/images/white_logo.png" alt="logo" />
+			{userRole === "freelancer" && (
```
Comment (lyudad) [inline libs/components/src/lib/dashboard-menu/dashboard-menu.tsx]: Снимок экрана 2022-11-28 в 20 31 25

File: apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx (hunk 5 above)
Comment (braulioportela79) [reply apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx]: Fixed for this page. I will fix the other pages on a different branch/PR.

File: apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx (hunk 3 above)
Comment (braulioportela79) [reply apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx]: Fixed.

File: libs/components/src/lib/dashboard-menu/dashboard-menu.tsx (hunk 6 above)
Comment (braulioportela79) [reply libs/components/src/lib/dashboard-menu/dashboard-menu.tsx]: Fixed.

File: apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx (hunk 4 above)
Comment (braulioportela79) [reply apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx]: Fixed. Custom hooks added.

**Pull request closed without merging**